src/qibocal/cli/auto_builder.py

```diff
@@ -74,13 +76,13 @@ def routine_name(self, routine, iteration):
         name = routine.replace("_", " ").title()
         return f"{name} - {iteration}"
 
-    def routine_qubits(self, routine_name, iteration):
+    def routine_qubits(self, task_uid: TaskId):
```

Comment:
Tiny simplification: in this context (same below), you can also just call it `task`. The type is already telling you that is a `TaskId`, so that's enough :)
```suggestion
    def routine_qubits(self, task: TaskId):
```